build.gradle

```diff
@@ -259,6 +266,97 @@ testClusters.integTest {
     }
 }
 
+String bwcVersion = "1.13.0.0";
+String baseName = "adMixedCluster"
+String bwcFilePath = "src/test/resources/org/opensearch/ad/bwc/"
+
+testClusters {
+    "${baseName}" {
+        testDistribution = "ARCHIVE"
+        versions = ["7.10.2","1.0.0"]
```

Comment:
We should have a way to define these versions as this is not scalable for each release.
We should open an issue for this, hopefully the plugin owners can help contribute it for us if not we can pick it up.

Reply:
Yeah this is temporary, we will definitely need a way to define versions for each release. I will open an issue for this.